mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule ExMarcel.MixProject do
     [
       app: :ex_marcel,
       version: "0.1.0",
-      elixir: "~> 1.12",
+      elixir: "~> 1.14",
```

Comment:
what will happen with apps running 1.12?

Reply:
Apps running Elixir below 1.14 probably won't work. That's should be fine, because only the latest three Elixir releases along with the latest 3 OTP releases should be supported.

Reply:
Tested it in an app with Elixir v1.12 OTP 24. It seems to be working. But there's a compilation warning
`warning: the dependency :ex_marcel requires Elixir "~> 1.14" but you are running on v1.12.3`.

Disclaimer - this is just my guess. I'm not 100% sure about this.
It may not be the case of this library, but Elixir Kernel and/or Erlang functions may differ. Relaxing the version may get someone using this library with older Elixir/OTP version and causing errors and/or crashes.